src/Console/Command/CheckCodingStandardsCommand.php

```diff
@@ -48,13 +48,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $dirname = !empty($input->getOption('directory')) ? $input->getOption('directory') : getcwd();
         // @codingStandardsIgnoreLine
         $exclude_dirs = !empty($input->getOption('exclude-dirs')) ? '--ignore=' . $input->getOption('exclude-dirs') . ' ' : '';
-        $standard = !empty($input->getOption('standard')) ? $input->getOption('standard') : $properties['phpcs-config'];
-        $basedir = $input->getOption('basedir');
+        $standard = !empty($input->getOption('standard')) ? $input->getOption('standard') : $properties['phpcs.config'];
+        $basedir = $input->getOption('project.basedir');
 
         //$width = !empty($input->getOption('width')) ? $input->getOption('width') : 80;
         $show = $input->getOption('show') ? true : false;
         ob_start();
-        $executable = $basedir . "/ssk/phpcs";
+        $executable = $basedir . "/toolkit/phpcs";
```

Comment:
```${toolkit.dir.bin}/phpcs``` instead, maybe more flexible

Reply:
Good idea, will see if I can do that. Need to simplify the property loading anyway. Load once before qa execution and make sure all commands have access to them.